samples/sample-rdk-rules/templates/lambda-permission.json

```diff
@@ -0,0 +1,15 @@
+
+{
+      "Type": "AWS::Lambda::Permission",
```

Comment:
Can you fix the indentation in these files?
The indentation level should follow the normal depth as if these files are standalone JSON files.